docs/topics/installing-web-console-or-cli-tool.adoc

```diff
@@ -38,7 +38,7 @@ endif::[]
 ====
 If you are installing on a Windows operating system:
 
-. Extract the `.zip` file to a folder named `mta` to avoid a `Path too long` error.
+. Extract the `.zip` file to a folder named `mta` to avoid a `Path too long` error _or_ extract the file with link:https://www.7-zip.org/download.html[7-Zip] to a folder of any name you choose.
 . If a *Confirm file replace* window is displayed during extraction, click *Yes to all*.
 ====
 +
```

Comment:
@RichardHoch 
Same as above - splitting the sentence instead of 'or'. 
In the Rules Development guide, you instruct the user to replace <MTA_HOME> with the actual path to the MTA installation. Shouldn't you say the same here? It is clearly needed in the next step of this procedure.

Reply:
Yes same instruction can be given in web console guide as well .

Reply:
Done.